client/components/tile-grid/README.md

```diff
@@ -0,0 +1,50 @@
+TileGrid (jsx)
+==========
+
+Component used to declare an area for displaying Tiles — it's the main wrapper that takes care of tile alignment and positioning.
+
+#### Props
+
+* `className`: Add your own class to the grid.
+* `isHidden`: Whether the entire grid should be hidden. Default is `false`.
+
+----------
+
+Tile (jsx)
+==========
+
+Component used to display a clickable tile with an image, call to action and description.
```

Comment:
Super-Nitpick: Oxford comma 😄

Reply:
Isn't that a subjective thing? 😕

Kidding, addressed in 0d15bddb51e4ee52235807ef91970b975b58b132.